src/wide-screen/components/MenuTabNav.js

```diff
@@ -20,6 +20,12 @@ const MenuTabList = styled.div`
   max-width: 100%;
   overflow: scroll;
   white-space: nowrap;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
```

Comment:
Nice lol - I was looking for ways to hide the horizontal scroll bar :))))) Thanks!